src/include/cephfs/libcephfs.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
 
 #define LIBCEPHFS_VER_MAJOR 10
 #define LIBCEPHFS_VER_MINOR 0
-#define LIBCEPHFS_VER_EXTRA 2
+#define LIBCEPHFS_VER_EXTRA 3
```

Comment:
Do we need to bump this version, given that we're just adding a new function? I suppose it doesn't hurt anything, but does this gain us anything?

Reply:
Ususlly in the open-iscsi/tcmu-runner project for both rbd and gluster handlers, they are always bumping the versions in the case like this. Then tcmu-runner can build the binary depending on this version to choose which API it should call. 

Here I assume we also need it for third part apps. Make sense ?